openjdk.build/makefile

```diff
@@ -159,7 +171,7 @@ ifneq (,$(findstring configure,$(MAKECMDGOALS)))
 else
   # Use wildcard and filter-out to find any directories which don't exist
   EXISTING_PREREQS_ROOT_LIST:=$(wildcard $(ABSOLUTE_PREREQS_ROOT_LIST))
-  #$(warning EXISTING_PREREQS_ROOT_LIST is $(EXISTING_PREREQS_ROOT_LIST))
+  $(warning EXISTING_PREREQS_ROOT_LIST is $(EXISTING_PREREQS_ROOT_LIST))
```

Comment:
Did you mean to leave this uncommented but not the similar statement at line 158?